dev.env

```diff
@@ -1,13 +1,19 @@
 DEBUG=1
+
 # HOST is currently used for absolut urls for example on return urls from oath
 # BIND_HOST is the host that flask is binding to when running as it's only server
 # BIND_HOST is ignored when running under waitress-serve
 HOST=localhost:5000
 BIND_HOST=0.0.0.0:5000
 HTTPS=0
 FLASK_SECRET_KEY=dev-secret-key
-DATABASE_URL=postgresql://localhost/bridge-server
-WEB3_PROVIDER_URI=http://localhost:8545
+
+DATABASE_URL=postgresql://docker:docker@localhost:5432/origin-bridge
+
+REDIS_URL=redis://127.0.0.1:6379/0
```

Comment:
Disclosure: I'm not familiar yet with the origin box setup...
Don't we run Redis in the docker container ?
If yes, should this be set to redis://docker:docker@localhost:6379/0 ?

Reply:
Redis doesn't require any credentials

Reply:
Yes, you are absolutely right !